mos_tests/conftest.py

```diff
@@ -434,3 +434,8 @@ def controller_remote(env):
 @pytest.fixture
 def openstack_client(controller_remote):
     return os_cli.OpenStack(controller_remote)
+
+
+def is_cinder_lvm_enabled(env):
```

Comment:
cinder_lvm can be enabled only if volumes_ceph is disabled. So you can use
`check_env('not(is_ceph_enabled)')`